en/intro-explore.md

```diff
@@ -0,0 +1,275 @@
+running on a fresh *libvirt* instance or real hardware, matching the 
+[minimum requirements][requirements], MAAS is installed with the following
+command:
+
+```bash
+sudo snap install --devmode --stable maas
+```
+
+While testing, both the *region controller* and the *rack controller* will run
+on this single machine, but you can have multiples of both to scale out MAAS to
+meet your requirements, from just a few machines to hundreds across different
+zones.
+
+To initialise MAAS for a single machine, issue the following command:
+
+```bash
+sudo maas init --mode all
+```
+
+After entering a username and password, web UI is now accessible from the
+configured address from where you can now login with your new credentials.
+
+See [Installation][installation] for further details.
+
+### Welcome to MAAS
+
+After a fresh MAAS installation, the web UI presents a
+[Welcome dashboard][welcome-page] to help set many system-wide options,
+including connectivity, image downloads, and authentication keys.
+
+
+<style>
+table th, table td {
+    background: #f7f7f7;
+    border: 0px solid;
+    padding: 15px 10px;
+}
+
+table.logos {
+    background: #f7f7f7;
+    border: 0px solid;
+    padding: 4px 4px;
+}
+
+table.logos th, table.logos td{
+    align="center";
+    valign="center";
+    border: 8px;
+    border-style: solid;
+    border-color: #ffffff;
+  }
+</style>
+
+<table width="500" border-width="0px" cellpadding="5">
+
+<tr>
+
+<td align="center" valign="center" border-width="0px" >
+<img src="../media/intro-explore__2.3_welcome1.png" alt="First page of Welcome settings" />
+<br />
+<i>Welcome</i> sets global values...
+</td>
+
+<td align="center" valign="center" border-width="0px">
+<img src="../media/intro-explore__2.3_welcome2.png" alt="Second page of Welcome settings" />
+<br />
+...before the MAAS <i>Dashboard</i>
+</td>
+
+</tr>
+
+</table>
+
+With this complete, you'll see *MAAS has been successfully set up*. Click 'Go
+to the Dashboard' to proceed (although you may need to wait a few moments for
+your selected images to sync locally).
+
+
+## Networking
+
+The *Dashboard* landing page, manually opened with a click of the MAAS logo,
+lists non-registered devices detected automatically on the network.  This
+[network discovery][device-disco] enables you to easily add or map devices
+already connected to your network that you may not necessarily want managed by
+MAAS.
+
+![network discovery][img__disco]
+
+Note:
+    Network discovery can be disabled at any time from the button on the
+    *Dashboard* view.
+
+### Spaces, fabrics, zones and subnets
+
+![MAAS network concepts][img__netconcepts]
+
+To accommodate rack installation within complex network topologies and the
+requirements of flexible machine deployment, MAAS offers comprehensive control
+over networking.
+
+Alongside the management of its own domain, configured via the DNS page of the
+web UI, MAAS can isolate machine deployment across fabrics, zones, fabrics and
```

Comment:
fabrics, fabrics and more fabrics. Shouldn't spaces be in there?

Reply:
thanks @evilnick!